### PR TITLE
added click listeners to elements of card view

### LINKED ISSUE #2472: feat add : redirect to fragment specific to Text View of tracks
**Expected Behaviour**

Attaching click listeners to the text views (inside card view) of tracks and redirecting to the corresponding fragment.
Ex- When Denis Nek will be clicked it would take the user to the speaker info fragment of Denis Nek
similarly selecting Dalton Hall will take user to all the sessions happening at Dalton hall

**Screenshots of the issue**

screenshot_20180517-105538

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by simarsingh24**
Fixes #2472 

Changes: added click listeners to the `Speaker Linear Layout` and `Location Linear Layout`

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Use selectable item background

**Comment by simarsingh24** [comment]
@iamareebjamal  please review

**Review by SIMRAN1 — COMMENTED** [review]
@simarsingh24 first rebase your branch with development using command git pull --rebase upstream development !

**Comment by simarsingh24** [comment]
@SIMRAN1 I am not sure it's needed, its mergeable and there are no conflicts!

**Comment by SIMRAN1** [comment]
Actually by seeing this *This branch is out-of-date with the base branch* I suggested you the rebase with development branch!!

**Comment by simarsingh24** [comment]
@iamareebjamal does it need any changes?

File: android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java [hunk 1]
```diff
@@ -130,6 +136,13 @@ protected void onResume() {
 
         handleVisibility();
     }
+    private void setUIColor (int color ){
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java]: Add line feed above

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java]: Space after color

File: android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerDetailsActivity.java [hunk 2]
```diff
@@ -182,33 +192,22 @@ private void loadSpeakerImage() {
...
             public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom from) {
                 progressBar.setVisibility(View.GONE);
-
                 speakerImage.setImageBitmap(bitmap);
-
-                Palette.from(bitmap).generate(paletteAsyncListener);
+                Palette.from(bitmap).generate(palette -> {
+                    Palette.Swatch swatch = palette.getDarkVibrantSwatch();
+                    int backgroundColor = ContextCompat.getColor(context, R.color.color_primary);
+                    if(swatch != null) {
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerDetailsActivity.java]: Soace after if

File: android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerDetailsActivity.java [hunk 3]
```diff
@@ -426,5 +432,5 @@ public void itemOnClick(Session session, int layoutPosition) {
         Intent intent = new Intent(this, SessionDetailActivity.class);
         startActivity(Views.openSessionDetails(session, intent));
     }
-
+
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerDetailsActivity.java]: Extra change

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java [hunk 4]
```diff
@@ -136,7 +136,8 @@ protected void onResume() {
 
         handleVisibility();
     }
-    private void setUIColor (int color ){
+    
+    private void setUIColor (int color){
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java]: Space after )

Please adhere to Java Style guidelines

File: android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java [hunk 5]
```diff
@@ -130,6 +136,14 @@ protected void onResume() {
 
         handleVisibility();
     }
+    
+    private void setUIColor (int color){
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/location/LocationActivity.java]: Space after )

**Comment by simarsingh24** [comment]
resolved

**Review by iamareebjamal — APPROVED** [review]

**Review by nikit19 — APPROVED** [review]

**Pull request merged**